packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -89,7 +92,19 @@ class SessionReplay implements SessionReplayEnrichmentPlugin {
     this.stopRecordingEvents = record({
       emit: (event) => {
         const eventString = JSON.stringify(event);
-        const shouldSplit = shouldSplitEventsList(this.events, eventString, this.maxPersistedEventsSize);
+        // Send the first two recorded events immediately
+        if (this.events.length === 1 && this.currentSequenceId === 0) {
+          this.sendEventsList({
+            events: this.events.concat(eventString),
+            sequenceId: this.currentSequenceId,
```

Comment:
Sorry that I'm just hooking into this PR for the config - if you don't mind, what is the sequenceId used for? Is it the batch?

Reply:
Gotcha - do we even end up with a batch Id?

Reply:
Nope, there's no batch id, just the sequence id